src/.clang-tidy

```diff
@@ -80,25 +78,29 @@ CheckOptions:
     value:           'NULL'
   - key:             modernize-use-default-member-init.UseAssignment
     value:           '1'
-  - key:   readability-identifier-naming.EnumCase
-    value: CamelCase
-  - key:   readability-identifier-naming.EnumConstantCase
-    value: CamelCase
-  - key:   readability-identifier-naming.ConstantMemberCase
-    value: CamelCase
+  # This is a *verification* check. The configuration **MUST** be kept in sync
+  # with .github/codechecker/config-verify.json
```

Comment:
I am sure I can create a CI check that validates that both are in sync. once this one is merged, i can take care of that (I did it for a few other things already, too. :) )